pt-br/dynamic-programming-pt.html.markdown

```diff
@@ -1,4 +1,4 @@
----
+﻿---
 category: Algorithms & Data Structures
```

Comment:
@LukaxFeh There seems to be a problem here. The deletion and addition of this line seems to break the YAML frontmatter needed for the page to build correctly. Please undo this change.